src/app/shared/components/validation-hint/validation-hint.component.spec.ts

```diff
@@ -23,4 +24,84 @@ describe('ValidationHintComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+  
+  describe('tests of checkInvalidText method', () => {
+    const errorsCasesMap = [
+      ['invalidSymbols', 'NAME_REGEX', NAME_REGEX, true, false, false, false, false],
+      ['invalidCharacters', 'NO_LATIN_REGEX', NO_LATIN_REGEX, false, true, false, false, false],
+      ['invalidStreet', 'STREET_REGEX', STREET_REGEX, false, false, true, false, false],
+      ['invalidHouse', 'HOUSE_REGEX', HOUSE_REGEX, false, false, false, true, false],
+      ['invalidSectionName', 'SECTION_NAME_REGEX', SECTION_NAME_REGEX, false, false, false, false, true],
+    ];
+
+    test.each(errorsCasesMap)('should assign to %s TRUE if required pattern equal to %s',
```

Comment:
`test.each` is a good way to test multiple variants, but a more clear way is to create a separate test for each condition